.travis.yml

```diff
@@ -15,11 +42,12 @@ script:
   - cargo build
   - cargo build --examples
   - cargo test
+  - BUILD_RELEASE="true" bash scripts/docker/build.sh
```

Comment:
I guess we only want to do this on builds in master branch?

Reply:
Correct.  Not sure how to do this in Travis YAML file, though...